src/lib/POWERMGNT/POWERMGNT.h

```diff
@@ -25,6 +25,18 @@
 #define MaxPower PWR_1000mW
 #define DefaultPowerEnum PWR_50mW
 
+#elif defined(TARGET_TX_BETAFPV_900_V1)
+#define MaxPower PWR_500mW
+#define DefaultPowerEnum PWR_100mW
```

Comment:
I think we want 50mW to be the default power across all TX targets?

Reply:
@wvarty I dont think it goes that low :|

Reply:
Oh, right. Well in that case the `switch` statement needs fixing so that 50mW isn't falling through to `default`